kubernetes/config-skeleton.libsonnet

```
@@ -69,6 +69,7 @@
         },
       },
 
+      podTolerantion: [],
```

Minor detail but I'd like to have this setting's name in the plural. 
```suggestion
      podTolerantions: [],
```
Mostly because other simple settings here are in plural like `podAnnotations` and `podLabels`. Also because the setting is an array, which implies plural.